spring-boot-project/spring-boot/src/main/java/org/springframework/boot/context/properties/source/ConfigurationPropertyName.java

```diff
@@ -448,7 +448,7 @@ private boolean defaultElementEquals(Elements e1, Elements e2, int i) {
 		int i2 = 0;
 		while (i1 < l1) {
 			if (i2 >= l2) {
-				return remainderIsNotAlphaNumberic(e1, i, i1);
+				return remainderIsNotAlphaNumeric(e1, i, i1);
```

Comment:
For consistency with `alpha-numeric -> alphanumeric`, this method should be renamed to `remainderIsNotAlphanumeric`.

Reply:
Done.